M2/InvertedIndex.py

```diff
@@ -6,14 +6,29 @@
 from enum import IntEnum
 import math
 from heapq import merge
+from simhash import Simhash, SimhashIndex
 
 class InvertedIndex:
     def __init__(self) -> None:
         self.inverted_index = SortedDict()
         self.inverted_index_files = []
+        self.simhashIndexes = SimhashIndex(list(), k=1)
 
     # Add a document to inverted index
     def addDocToInvertedIndex(self, docId, tokens, important_words_set, important_words_tags):
+        # One possible issue with approach is that we might have docIds in the hashmap in assignment3_m1.py
```

Comment:
I guess if there's no information on deleted pages in the inverted index, it doesn't matter

Reply:
Yeah, I guess. It should be okay.

Reply:
I will just merge it then.